src/pages/CreateErrata.vue

```diff
@@ -561,7 +561,11 @@
                       artifact.name.includes('.src.')
                     )
                       continue
-                    let pkg = splitRpmFileName(artifact.name)
+                    let pkg = artifact.meta
+                    // TODO: Remove this fallback afte some time
+                    if (!pkg) {
+                      pkg = splitRpmFileName(artifact.name)
+                    }
                     pkgs.push({
                       name: pkg.name,
                       epoch: 0,
```

Comment:
Shouldn't the obtained epoch be used here?

Reply:
Nice catch! Updated :)